webapi/src/test/scala/org/knora/webapi/e2e/v2/ResourcesRouteV2E2ESpec.scala

```diff
@@ -2401,6 +2232,176 @@ class ResourcesRouteV2E2ESpec extends E2ESpec(ResourcesRouteV2E2ESpec.config) {
       val editValueResponseDoc            = responseToJsonLDDocument(editValueResponse)
       assert(editValueResponse.status == StatusCodes.OK, responseToString(editValueResponse))
     }
+
+    "correctly load and request resources that have a isSequenceOf relation to a video resource" in {
```

Comment:
Same here for positive vs negative testing.

Reply:
as above